SPECS/ed/ed.spec

```diff
@@ -1,16 +1,14 @@
 Summary:        Ed - A line-oriented text editor
 Name:           ed
-Version:        1.14.2
-Release:        9%{?dist}
+Version:        1.20
+Release:        1%{?dist}
 URL:            https://www.gnu.org/software/ed/
 License:        GPLv3
 Group:          Applications/System
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
-# Official source under https://ftp.gnu.org/gnu/ed/ed-1.14.2.tar.lz.
-# We don't have lzip to decompress it.
-Source0:        https://src.fedoraproject.org/repo/pkgs/%{name}/%{name}-%{version}.tar.xz/sha512/de838a6df785c7dc80f4b5ba84330bbe743983fd81218321d4ab84c4c3688fdafb4c005502f3228f0bfa2b6bcf342d64d9523ab73ee440b4f305a033f567cbc2/%{name}-%{version}.tar.xz
-
+# We don't have lzip to decompress it. Hence converted to tar.gz
```

Comment:
🥇 
Maybe once the churn settles down, we could promote lzip from extended -> core and use upstream sources :)